dotcom-rendering/src/web/lib/contributions.ts

```diff
@@ -246,3 +247,23 @@ export const lazyFetchEmailWithTimeout =
 export const getContributionsServiceUrl = (
 	CAPIArticle: CAPIArticleType,
 ): string => process.env.SDC_URL ?? CAPIArticle.contributionsServiceUrl;
+
+type PurchaseInfo = HeaderPayload['targeting']['purchaseInfo'];
```

Comment:
can we import the `PurchaseInfo` type here?
https://github.com/guardian/support-dotcom-components/blob/f2b9bbc7f4abb7d0386d4a33d10ae525aeedd124/packages/shared/src/types/targeting/header.ts#L5

Reply:
Would need to be exported here also: https://github.com/guardian/support-dotcom-components/blob/main/packages/dotcom/src/types.ts